12_MINI/error-codes.yaml

```diff
@@ -215,6 +215,11 @@ Errors:
   text: "ESP doesn't seem to be connected."
   id: "ESP_NOT_CONNECTED"
   approved: false
+- code: "12601"
+  title: ""
+  text: "Unrecognised error occured."
+  id: "GENERAL_ERROR"
+  approved: false
```

Comment:
What error is this for? Isn't it that you're always handling a concrete error in the code, why wouldn't you then always raise a specific error instead of this "unknown" one?

Reply:
There is a find function that searches through errors and if the error code is unrecognisable, it falls to this error.

Right now there is no instance, where this would occur (thats why it was not in yamls before). But is behaves as a fail safe for future mistakes made by programmers.

Jacob insisted that any error code (used or unused) should be included in the yamls.

Reply:
The funky thing is that the error lookup is actually compile-time, so the proper handling of not finding an error should be a compilation error... But I once spent a good hour trying to make it work that way and didn't succeed (IIRC it was a limitation of `constexpr` that should be lifted in one of the recent standards, but forgot the details).

It would still be very much preferable to somehow make it fail at compile time, could you maybe try to have another look at that? If you still can't find a way, as a last resort using this, a better id would probably be `UNKNOWN_ERROR` or maybe `NOT_FOUND_ERROR`?